app/components/bulk_update/trainee_uploads/row/view.rb

```diff
@@ -13,6 +13,18 @@ class View < ViewComponent::Base
           "cancelled" => "govuk-tag--yellow",
         }.freeze
 
+        PATHS = {
+          "succeeded" => lambda do |upload|
+            Rails.application.routes.url_helpers.bulk_update_trainees_details_path(upload)
+          end,
+          "in_progress" => lambda do |upload|
+            Rails.application.routes.url_helpers.bulk_update_trainees_submission_path(upload)
+          end,
+          "failed" => lambda do |upload|
+            Rails.application.routes.url_helpers.bulk_update_trainees_review_error_path(upload)
+          end,
+        }.freeze
```

Comment:
I think you can do this https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/components/sort_by/view.rb#L5

and then

```suggestion
        PATHS = {
          "succeeded" => lambda do |upload|
            bulk_update_trainees_details_path(upload)
          end,
          "in_progress" => lambda do |upload|
            bulk_update_trainees_submission_path(upload)
          end,
          "failed" => lambda do |upload|
            bulk_update_trainees_review_error_path(upload)
          end,
        }.freeze
```

Wondering do we really need to use a lamda?

Reply:
To use the paths by including the `url_helpers` the calls to the paths have to be in the instance level not the class level where the constant is defined. We can do that if that's the preference and then the lamdas won't be needed.